README.md

```diff
@@ -1,9 +1,127 @@
+# #comfortable-echidna @breyana @zezemanolo 
 # Multiple Paradigms
 
+https://github.com/GuildCrafts/web-development-js/issues/124
 Markdown parser implementations using three different programming paradigms: **object-oriented**, **functional**, and **imperative**.
 
 Base repository for the [Multiple Paradigms](https://github.com/GuildCrafts/web-development-js/issues/124) goal.
 
 ## Installation and Setup
 
+Work in progress: user will only need to run `npm i` from root directory to install dependencies. User will run `npm test` on terminal to test implementations.
+
 ## Usage and Examples
+
+Implement a Markdown parser using three different programming paradigms: **object-oriented**, **functional**, and **imperative**.
+
+Fork the [multiple-paradigms][multiple-paradigms] repository and use the fork as your project artifact.
+
+The repository includes a skeleton script at `bin/parse` where you need to load and execute your Markdown parsers for each programming paradigm.
+
+To parse a file, use the `npm run parse:<paradigm> <file.md>` command, where `<paradigm>` is one of either `oo` (object-oriented), `functional`, or `imperative` and `<file.md>` is a path to a Markdown-formatted file.
+
+For example, say we have a file `hello.md` with the contents:
+
+```shell-session
+$ cat hello.md
+_Hello_.
+
+- My name is **Inigo Montoya**.
+- You killed my father.
+- Prepare to die.
+```
+
+...then the command `npm run parse:oo hello.md` (or `parse:functional`, or `parse:imperative`) should print the following:
+
+```shell-session
+$ npm run parse:oo hello.md
+<p><em>Hello</em>.</p>
+<ul>
+<li>My name is <strong>Inigo Montoya</strong>.</li>
+<li>You killed my father.</li>
+<li>Prepare to die.</li>
+</ul>
+```
+
+## Context
+
+_Why is this goal important? How is it useful? What questions will it raise?_
+
+Knowing how, when, and where to apply a particular _programming paradigms_ to a a piece of code is a key skill of any self-respecting software developer.
+
+Being fluent in different paradigms will help you to think about and understand software architecture, and to make better design choices. It is also a crucial tool for communicating effectively with other developers.
+
+
+
+## Specifications
+
+_List of specifications (specs) for the completed goal. These are declarative sentences (statements) describing a feature of the final product._
+
+**This is a core goal. The specifications are non-negotiable. To complete this goal, you must complete all specs listed below.**
+
+- [ ] Artifact produced is a fork of the [multiple-paradigms][multiple-paradigms] repo.
+- [ ] Can run all tests with `npm test`.
+- [ ] All tests are passing.
```

Comment:
@jrob8577 , added specs around test coverage and removed stretch goals. Can we merge to master?